virttest/utils_net.py

```diff
@@ -3402,29 +3402,25 @@ def _process_output(cmd, reg_pattern):
                                "mac %s" % mac_address)
 
 
-def update_mac_ip_address(vm, params, timeout=240):
+def update_mac_ip_address(vm, timeout=240):
     """
     Get mac and ip address from guest then update the mac pool and
     address cache
 
-    :param vm: VM object
-    :param params: Dictionary with the test parameters.
+    :param vm: VM object.
+    :param timeout: Time (seconds) to keep trying to log in.
     """
     try:
         session = vm.wait_for_serial_login(timeout=timeout)
         addr_map = get_guest_address_map(session)
         session.close()
-        if len(addr_map) != len(vm.virtnet):
```

Comment:
The test cases with multiple interface need this check, one acceptance case might be affected, will have a test

Reply:
@waynesun09 ok, I see. If we do need this check, it must be well-defined because we have the situation that the NIC was hotplugged and it had no configuration file.

BTW, the current check would always be failed because one NIC may have both ipv4 and ipv6 addresses and the structure of `addr_map` is:
```python
addr_map = {
    '$MAC': '...',    # ipv4 address
    '$MAC_6': '...',  # ipv6 address
    ...
}
```